.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        REACT_DIST: [16, 17]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          cache: 'npm'
+      - run: yarn
+      - run: yarn add react@${{ matrix.REACT_DIST }} react-dom@${{ matrix.REACT_DIST }}
+      - run: yarn --cwd www
+      - run: yarn test
+      - run: npm run build
+      - run: npm run semantic-release
```

Comment:
I guess this wouldn't work on GitHub Actions because this requires tokens like `NPM_TOKEN` and `GITHUB_TOKEN`, which would be set on TravisCI.

```
    env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Unfortunately, I don't have permission to set up `semantic-release`. 
@jquense Could you help us?

Reply:
Yes, I forgot to mention that an admin needs to setup secrets.

Reply:
What about removing the script?
Personally, I think it's good to release a new version manually instead of doing it on CI. I don't have the permission though.

Reply:
I'd like to introduce some breaking changes like dropping IE support and removing PropTypes in the future. In that time, I'd like to release the breaking changes as a single major version, not separately.
But we can't do that because this script releases a new version automatically, so I'd like to remove running `semantic-release` on CI.

Reply:
We can collect breaking changes on an `alpha` branch and merge that once it's ready. `semantic-release` will create an `alpha` release channel from that branch for test. We're doing that for `@testing-library/react` and it has been working great so far.